packages/agreement-process/src/services/agreementSubmissionProcessor.ts

```diff
@@ -148,7 +148,6 @@ const submitAgreement = async (
 
   const agreements = (
     await agreementQuery.getAllAgreements({
-      producerId: agreement.producerId,
       consumerId: agreement.consumerId,
       eserviceId: agreement.eserviceId,
       agreementStates: [agreementState.active, agreementState.suspended],
```

Comment:
Unrelated, but reviewing the submission logic I found this difference with the Scala code